### PR TITLE
refactor(odyssey-react): tokens for Toast theme

### PR DESCRIPTION
**Pull request by andrewberg-okta**

### CODE REVIEW

File: packages/odyssey-react/src/components/Toast/Toast.module.scss [hunk 1]
```diff
@@ -62,7 +62,7 @@
 
 .icon {
   width: var(--IconSize);
-  color: var(--IconColor);
+  color: var(--IconTextColor);
```
Comment (jeffbelser-okta) [inline packages/odyssey-react/src/components/Toast/Toast.module.scss:65]: wouldn't this just be `IconColor` in the same way we use `font-size` declarations but create tokens name `IconSize` ? Perhaps you can check and see what patterns are more prevalent in the code base for us now

**Review by jeffbelser-okta — APPROVED** [review]
one note about `IconTextColor` inline here

File: packages/odyssey-react/src/components/Toast/Toast.module.scss (hunk 1 above)
Comment (andrewberg-okta) [reply packages/odyssey-react/src/components/Toast/Toast.module.scss:65]: This is the only place that we set it specifically, with color passing linting seems safe to use the more condensed version.

Comment (andrewberg-okta) [reply packages/odyssey-react/src/components/Toast/Toast.module.scss]: addressed in 671cd3a

**Pull request merged**